lessons/8_loops/index.html

```diff
@@ -0,0 +1,308 @@
+                console.log(items[0]);
+
+                const cars = [{
+                    name: "Fiat",
+                    model: "IV-MODEL",
+                    color: "white" 
+                },{
+                    name: "Mercedes-Benz",
+                    model: "MOTORWAGEN 1886",
+                    color: "red"
+                }];
+                console.log('Car:', cars[1]);
+                console.log('Car Name:', cars[1].name);
+            </code></pre>
+        </section>
+
+        <section>
+          <h5>
+            Now, you need to show all the items that you have store in your array
+          </h5>
+          <pre
+            class="fragment"
+          ><code data-trim  class="hljs">
+                const items = ["Item 01", "Item 02", "Item 03", "Item 04"];
+                console.log(items[0]);
+                console.log(items[1]);
+                console.log(items[2]);
+                console.log(items[3]);
+            </code></pre>
+        </section>
+
+        <section>
+          <h5>
+            BUT... what happend if the array have 30 or even more items 🤯
+          </h5>
+          <img src="./noo.gif" width="180" height="130" />
+          <br />
+          <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 1", "Item 2", "Item 3", ..., "Item 30"];
+                console.log(items[0]);
+                console.log(items[1]);
+                .
+                .
+                .
+                console.log(items[29]);
+            </code></pre>
+        </section>
+
+        <section>
+          <h3>Don't worry exist a better solution for that!! 🎉</h2>
+          <h1 class="fragment">LOOPS</h3>
+        </section>
+
+        <section>
+            <h3>What is Loops?</h3>
+          <ul>
+            <li>
+             Loops is mainly used for iterating over arrays or objects if you have to run the same code several times.
+            </li>
+             
+          </ul>
+        </section>
+  
+        <section>
+            <pre><code data-trim class="" >
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                //to print the items always is running items[0] -> just the index is changing
+                console.log(items[0]);  // index: 0
+                console.log(items[1]);  // index: 1
+                console.log(items[2]);  // index: 2
+                console.log(items[3]);  // index: 3
+                console.log(items[4]);  // index: 4
+                console.log(items[5]);  // index: 5
+                console.log(items[6]);  // index: 6
+            </code></pre>
+            <br/>
+            <h2 class="fragment">Solution</h2>
+            <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                for (let i = 0; i < items.length; i++) {
```

Comment:
Just realised that they don't know what the condition is...But should be fine.

Reply:
we can explain that during class.